server/sonar-web/src/main/js/components/SourceViewer/SourceViewerHeader.tsx

```diff
@@ -75,11 +78,10 @@ export default class SourceViewerHeader extends React.PureComponent<Props, State
     } = this.props.sourceViewerFile;
     const isUnitTest = q === 'UTS';
     const workspace = false;
-    let rawSourcesLink =
-      getBaseUrl() + `/api/sources/raw?key=${encodeURIComponent(this.props.sourceViewerFile.key)}`;
-    if (this.props.branch) {
-      rawSourcesLink += `&branch=${encodeURIComponent(this.props.branch)}`;
-    }
+    const rawSourcesLink =
+      getBaseUrl() +
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Use a template literal instead of concatenating with a string literal. ([typescript:S3512](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS3512))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&pullRequest=3010&issues=AWIFEiEyJ6stENANvNAK&open=AWIFEiEyJ6stENANvNAK)

[ISSUE_KEY]: AWIFEiEyJ6stENANvNAK